.github/workflows/release.yml

```diff
@@ -14,6 +14,10 @@ on:
   push:
     tags:
       - 'v*'
+
+permissions:
+  contents: write
```

Comment:
Can you add this to the individual job, rather than the entire workflow? Also, since you're specifying an allowlist of perms, it'll drop all other perms. Are there any other read perms this workflow will need?

Reply:
Great callout 👍 We need to be able to checkout the repo but that should be included in `contents: write` (because it extends `read`)